### DISCUSSION TITLE
is there any way to realize bg top/bottom？

### DISCUSSION THREAD

**Discussion by Tasselmi** [marp-team/marp#354] [Q&A] [answered]
well， I do know to use something like `![bg left:25%](images/orange.png)`
Is there any way to put background picture on top or on bottom?

Thanks.

**Comment by yhatt** [comment, marked as answer]
There is _not yet_.

But probably we may consider about `top` and `bottom` keyword for splitted backgrounds if there are many requests for that.

I think that is especially useful for the title slide, like that:

```markdown
---
class: invert
paginate: true
---

<!--
NOTE: This is just an idea for new vertical split keyword about Marpit's split
background. `top:60%` is not working in reality. 
-->

![bg top:60%](https://images.unsplash.com/photo-1622004582495-46ab3d5e35af)

# What a cool title slide

This is a *proposal*, and not working for now.
```

![](https://user-images.githubusercontent.com/3993388/191982059-66bcde55-a7da-4973-9554-cd0229e200ae.png)

**Reply by Tasselmi** [reply]
Thanks for your quick reply. Hope will implement in the future.
